lib/datasource/common.ts

```diff
@@ -17,6 +17,7 @@ export type DigestConfig = Config;
 export interface Release {
   changelogUrl?: string;
   gitref?: string;
+  gitRef?: string;
```

Comment:
@rarkins we should only use one of them.

Reply:
It should be `gitRef`

Reply:
fixing in seperate pr?

Reply:
sure

Reply:
done